src/pybind/mgr/dashboard/frontend/src/app/shared/components/grafana/grafana.component.ts

```diff
@@ -78,28 +206,26 @@ export class GrafanaComponent implements OnInit, OnChanges {
     this.settingsService
       .validateGrafanaDashboardUrl(this.uid)
       .subscribe((data: any) => (this.dashboardExist = data === 200));
-    this.url = this.baseUrl + this.uid + '/' + this.grafanaPath + '&refresh=2s' + this.mode;
+    this.url =
+      this.baseUrl +
+      this.uid +
+      '/' +
+      this.grafanaPath +
+      '&refresh=2s' +
```

Comment:
Shouldn't we allow the user to select the "refresh" too?

Reply:
@ricardoasmarques planning to add this setting in a configuration page later http://tracker.ceph.com/issues/35809, because user might not be changing refresh interval much.

Reply:
Agree, let's reduce the number of controls for better usability. Thanks.